cirq-google/cirq_google/engine/engine_client.py

```diff
@@ -395,14 +395,16 @@ async def create_job_async(
         run_name: str = "",
         device_config_name: str = "",
     ) -> Tuple[str, quantum.QuantumJob]:
-        """Creates and runs a job on Quantum Engine.
+        """Creates and runs a job on Quantum Engine. Either both `run_name` and
```

Comment:
Following the Python docstring conventions, the first line of docstring should be a one-line summary, followed by a blank line, and the more elaborate description: https://peps.python.org/pep-0257/#multi-line-docstrings

Ditto elsewhere